editor/editor_node.cpp

```diff
@@ -323,9 +324,18 @@ void EditorNode::_notification(int p_what) {
 
 	if (p_what == MainLoop::NOTIFICATION_WM_FOCUS_IN) {
 
+		// Restore the original FPS cap after focusing back on the editor
+		OS::get_singleton()->set_low_processor_usage_mode_sleep_usec(int(EDITOR_GET("interface/editor/low_processor_mode_sleep_usec")));
+
 		EditorFileSystem::get_singleton()->scan_changes();
 	}
 
+	if (p_what == MainLoop::NOTIFICATION_WM_FOCUS_OUT) {
+
+		// Set a low FPS cap to decrease CPU/GPU usage while the editor is unfocused
+		OS::get_singleton()->set_low_processor_usage_mode_sleep_usec(int(EDITOR_GET("interface/editor/unfocused_low_processor_mode_sleep_usec")));
```

Comment:
What if `low_processor_usage_mode_sleep_usec` was already enabled but the setting is disabled? This line would turn off `low_processor_usage_mode_sleep_usec`.

Reply:
The `low_processor_mode_sleep_usec` setting is independent from `low_processor_mode` (the former accepts an integer, the latter accepts a boolean). Note that having low-processor mode enabled with a `low_processor_mode_sleep_usec` set to 0 is not the same as having it disabled. In this case, framerate will be unlimited, but the application will not be re-rendered if nothing changes in the viewport.